### DISCUSSION TITLE
What is the codec string for PCM?

### DISCUSSION THREAD

**Discussion by AlexVestin** [w3c/webcodecs#792] [Q&A] [answered]
Hi, thanks for this amazing project. 

I've been wanting to use the pass-through pcm audio decoder, described here https://www.w3.org/TR/webcodecs-pcm-codec-registration/#audiodecoderconfig-description . FFmpeg reports the codec_id as pcm_s16le, so I tried to use "pcm-s16" as the codec string, but I get the error "Unknown or ambiguous codec string" when configuring the decoder, what is the correct codec string to use here?

**Comment by sandersdan** [comment, marked as answer]
`pcm-s16` is the correct codec string, but I don't think it is implemented in any browser (at least there are no WPT tests and Chrome doesn't implement it). As an alternative, you should be able to directly construct an `AudioData` from the same bytes.

**Reply by AlexVestin** [reply]
Constructing the AudioData directly worked and was pretty straightforward, thanks!
